File: language-examples/js-hello-world/hello.js
/*
    Tick
    Trick
    Track
*/
class Temp {
    boo(){

    }
}

function OldSchool(){

}

OldSchool.prototype.boo = function (){

};

function foo(x){
    let foo = 4;
    if(foo){
        noop();
        noop();
    }else if(foo) {
        nesting();
    }else if(foo) {
        nesting();
    } else {
        return "jo!";
    }
    // Js comment
    let str = `
        Tick
        Trick
        Track
    `;

    let swotch = {
        1:'a',
        2:'e',
        3:'i',
        4:'o',
        5:'u'
    }
    for(let i = 0; i < 5; i++){
        handle(i);
    }
}

function noop(){
    "";
}

function nesting(){
    function nested(){
        const noop = () => { ""; };
    }
    console.log("hi");
}

function handle(num){
    console.log("hi");
}

console.log("hi");

// new stuff

File: language-examples/python-hello-world/hello.py
"""
    Tick
    Trick
    Track
"""
def Foo(x):
    foo = 4
    if foo:
        noop()
    # Python comment
    str = """
        Tick
        Trick
        Track
    """
    switch={
          1:'a',
          2:'e',
          3:'i',
          4:'o',
          5:'u'
    }
    for i in range(1, 5):
        handle(i)

def noop():
    ""

def nesting():
    def nested():
        def noop():
            ""

def handle(num):
    print num

print Foo("hi")

